### PR TITLE
Support for Arm64

### PR DESCRIPTION
**Pull request by joehoyle**

### CODE REVIEW

File: inc/composer/class-docker-compose-generator.php [hunk 1]
```diff
@@ -653,6 +653,15 @@ protected function get_config( $module = 'local-server' ) : array {
 		return array_merge( $defaults, $config );
 	}
 
+	/**
+	 * Check if the current system is ARM64.
+	 *
+	 * @return bool
+	 */
+	protected function is_arm64() : bool {
+		return strpos( php_uname(), 'aarch64' ) !== false;
```
Comment (roborourke) [inline inc/composer/class-docker-compose-generator.php:662]: I think you can pass `'m'` to `php_uname()` to access this value directly eg:
```suggestion
		return php_uname( 'm' ) === 'aarch64';
```

File: inc/composer/class-docker-compose-generator.php [hunk 2]
```diff
@@ -375,7 +375,7 @@ protected function get_service_kibana() : array {
 	protected function get_service_s3() : array {
 		return [
 			's3' => [
-				'image' => 'minio/minio:RELEASE.2020-03-19T21-49-00Z',
+				'image' => 'minio/minio:RELEASE.2020-12-29T23-29-29Z',
```
Comment (roborourke) [inline inc/composer/class-docker-compose-generator.php:378]: Guessing this is back compatible?

**Review by roborourke — COMMENTED** [review]
Looks like we'll need to cut new / updated images for all the services we've made including tachyon and the analytics containers? Cheers for getting a head start on this.

File: inc/composer/class-docker-compose-generator.php (hunk 2 above)
Comment (joehoyle) [reply inc/composer/class-docker-compose-generator.php:378]: Yeah not sure tbh. Damn these date version numbers

File: inc/composer/class-docker-compose-generator.php [hunk 3]
```diff
@@ -196,7 +196,7 @@ protected function get_service_cavalcade() : array {
 	protected function get_service_nginx() : array {
 		return [
 			'nginx' => [
-				'image' => 'humanmade/altis-local-server-nginx:3.1.0',
+				'image' => 'humanmade/altis-local-server-nginx:3.1.0' . ( $this->is_arm64() ? '-arm64' : '' ),
```
Comment (joehoyle) [inline inc/composer/class-docker-compose-generator.php:199]: Todo: let's not merge until we have multi-arch images

**Comment by ntwb** [comment]
I've created #343 with all native ARM64 images running on Altis v8 with no emulation

**Comment by roborourke** [comment]
Superceded by #343 and multiarch builds

**Pull request closed without merging**